### ISSUE TITLE
tarpaulin installation is broken

### ISSUE THREAD

**Issue by jplatte** [taiki-e/install-action#167]
Since today, the following CI step is consistently [failing](https://github.com/matrix-org/matrix-rust-sdk/actions/runs/5680145335/job/15393680800?pr=2340):

```yml
    - name: Install tarpaulin
      uses: taiki-e/install-action@v2
      with:
        tool: cargo-tarpaulin
```

with the error

```
info: verifying sha256 checksum for cargo-tarpaulin-x86_64-unknown-linux-musl.tar.gz
sha256sum: WARNING: 1 computed checksum did NOT match
```

**Comment by taiki-e** [comment]
It seems the tarpaulin maintainer re-compiled binaries for 0.26.0.

- first (1 month ago): https://github.com/xd009642/tarpaulin/actions/runs/5293372127
- second (3 hours ago): https://github.com/xd009642/tarpaulin/actions/runs/5678274297


Looking at [the issue](https://github.com/xd009642/tarpaulin/issues/1341), it appears that the 0.26.0 release workflow was also accidentally retriggered when addressing a problem with the 0.26.1 release.

**Comment by taiki-e** [comment]
Fixed in 2.12.19. Thanks for the report!
